src/compiler/parser.ts

```diff
@@ -2243,7 +2260,13 @@ namespace ts {
                 isStartOfType(/*inStartOfParameter*/ true);
         }
 
-        function parseParameter(requireEqualsToken?: boolean): ParameterDeclaration {
+        function parseParameterNoSpeculation(): ParameterDeclaration {

```

Comment:
Is this wrapper necessary, since the default behavior is the same as `parseParameter()`?

Reply:
Removed `parseParameter`, now should always use this or `parseParameterInSpeculation`.